pkg/kubelet/dockertools/manager.go

```diff
@@ -1552,7 +1552,7 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	containerLogFile := containerInfo.LogPath
 	symlinkFile := LogSymlink(dm.containerLogsDir, kubecontainer.GetPodFullName(pod), container.Name, id.ID)
 	if err = dm.os.Symlink(containerLogFile, symlinkFile); err != nil {
-		glog.Errorf("Failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
+		glog.Errorf("failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
```

Comment:
I checked the various .md files in the repo and couldn't find anything. I know that error messages must start with a lowercase letter.